incubator/nodejs-kafka/templates/default/app.js

```diff
@@ -0,0 +1,115 @@
+const express = require('express');
+
+module.exports = (options) => {
+  // options.kafka is a required node-rdkafka:
+  // - api docs: https://blizzard.github.io/node-rdkafka/current/
+  // - quick start: https://github.com/Blizzard/node-rdkafka#usage
+  const Kafka = options.Kafka;
+
+  // options.log is a pino logger:
+  // - https://github.com/pinojs/pino/blob/master/docs/api.md#logger
+  const log = options.log;
+
+  // options.app is an express instance, routes can be configured.
+  const app = options.app;
+
+  // KAFKA_BOOTSTRAP_SERVERS can be injected by the Service Binding Operator
+  // - https://github.com/redhat-developer/service-binding-operator
+  const brokers = process.env.KAFKA_BOOTSTRAP_SERVERS || 'localhost:9092'
```

Comment:
I can bake it in and expose it as `options.brokers`. It seemed like the name might be an ephemeral detail of WIP examples.  "Servers" are called "brokers" by Kafka, and I've no idea what "bootstrap" means in this context. Could we call it `KAFKA_BROKERS` (Kafka's term) or `KAFKA_SERVICES` (k8s' term)? If not, `KAFKA_BOOTSTRAP_SERVERS` it can be.

Reply:
Yeah I think this is a short-term stop gap while the details of how Service Binding works are hammered out.

For example we'll need to support the binding information that Event Streams uses, which is not just host:port but also TLS-enabled and with authentication via SASL-SCRAM.

One simple env var is OK now (any maybe we do leave it in the template) and we can revisitit in the future.